### PR TITLE
NO-ISSUE: Bumping ocm-sdk version.

### PR DESCRIPTION
**Pull request by ybettan**
The current version contains a security vulnerability.

Signed-off-by: Yoni Bettan <ybettan@redhat.com>

### CODE REVIEW

File: pkg/ocm/client.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func (l *SdKLogger) Error(ctx context.Context, format string, args ...interface{
 	l.FieldLogger.Errorf(format, args...)
 }
 
+func (l *SdKLogger) Fatal(ctx context.Context, format string, args ...interface{}) {
```
Comment (filanov) [inline pkg/ocm/client.go:74]: Where it is used? fatal log with crash the service.

Comment (ybettan) [reply pkg/ocm/client.go:74]: It is not but it was added in the logger interface used in ocm-sdk so in order to use our current logger we need to implement it so we can use it as the required interface.
@filanov

**Review by filanov — APPROVED** [review]

**Comment by ybettan** [comment]
Thanks @filanov

**Pull request merged**